src/main/java/com/wavefront/internal/reporter/WavefrontInternalReporter.java

```diff
@@ -180,6 +182,14 @@ public WavefrontInternalReporter build(WavefrontSender wavefrontSender) {
   private final String source;
   private final Map<String, String> reporterPointTags;
   private final Set<HistogramGranularity> histogramGranularities;
+  private final WavefrontSdkMetricsRegistry sdkMetricsRegistry;
+
+  private final WavefrontSdkCounter gaugesReported;
+  private final WavefrontSdkCounter countersReported;
+  private final WavefrontSdkCounter histogramsReported;
```

Comment:
can you break this down into nonWavefrontHistogramReported and WavefrontHistogramReported?

Reply:
agree here. lgtm otherwise.